api/v2/Routes/isadmin/index.js

```diff
@@ -7,13 +7,25 @@ const router = Router({ mergeParams: true });
 
 // Responds with whether or not the current user is an admin
 router.get('/', async (req, res) => {
-    const authHeader = req.headers['authorization'];
-    if (!authHeader) {
-        throw new AuthorizationError("Authorization Header is empty.");
+    let adminStatus = false;
+    try {
+        const authHeader = req.headers['authorization'];
+        if (!authHeader) {
+            throw new AuthorizationError("Authorization Header is empty.");
+        }
+        const authEmail = await getEmailFromAuth(authHeader);
+        adminStatus = await isAdmin(authEmail);
+    } catch (error) {
```

Comment:
`error` tends to be a protected verb, so we tend to prefer going with `err` or something along those lines.